src/components/views/voip/PipView.tsx

```diff
@@ -54,6 +54,11 @@ const SHOW_CALL_IN_STATES = [
     CallState.WaitLocalMedia,
 ];
 
+type CreatePipContent = (object: {
```

Comment:
might be good to give this a comment saying what this is and what the properties in the function parameter mean. (I couldn't figure out where the function is called?)